olefile/olefile.py

```diff
@@ -188,9 +178,6 @@ def get_logger(name, level=logging.CRITICAL+1):
         return logger
     # get a new logger:
     logger = logging.getLogger(name)
-    # only add a NullHandler for this logger, it is up to the application
-    # to configure its own logging:
-    logger.addHandler(NullHandler())
```

Comment:
I think we still need to add a null handler, i.e. logging.NullHandler: see https://docs.python.org/2/howto/logging.html#library-config

Reply:
Added back, via reset and forced push.